dotcom-rendering/src/components/TagFrontHeader.tsx

```diff
@@ -0,0 +1,279 @@
+import { css } from '@emotion/react';
+import {
+	breakpoints,
+	from,
+	headline,
+	neutral,
+	space,
+	until,
+} from '@guardian/source-foundations';
+import { decideContainerOverrides } from '../lib/decideContainerOverrides';
+import type { DCRContainerPalette } from '../types/front';
+import { ContainerTitle } from './ContainerTitle';
+import { generateSources } from './Picture';
+
+type Props = {
+	title: string;
+	description?: string;
+	image?: string;
+	containerPalette?: DCRContainerPalette;
+};
+
+const width = (columns: number, columnWidth: number, columnGap: number) =>
+	`width: ${columns * columnWidth + (columns - 1) * columnGap}px;`;
+
+/** Not all browsers support CSS grid, so we set explicit width as a fallback */
```

Comment:
Do browser that do not support `grid` support the `@supports not` query?

Reply:
Not sure to be honest, I copied this from the [FrontSection](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/components/FrontSection.tsx#L82) component, so it might be worth a wider investigation